molecule/default/verify.yml

```diff
@@ -8,3 +8,4 @@
       ansible.builtin.assert:
         that:
           - ansible_facts.services["keycloak.service"]["state"] == "running"
+          - ansible_facts.services["keycloak.service"]["status"] == "enabled"
```

Comment:
We need to add that in the validation role in the other collection!!! (also we need to turn this verify into a validation role for keycloak)